other/diffeq/mnist_odenet.jl

```diff
@@ -37,15 +37,29 @@ ps = Flux.params(dudt)
 function n_ode(batch_u0, batch_t)
     neural_ode(dudt,batch_u0,(0.,25.),Tsit5(),
                save_start=false,
-               saveat=batch_t, reltol=1e-3,abstol=1e-3)
+               saveat=batch_t,      # Ugly way to only get sol[end] 
+               reltol=1e-3,abstol=1e-3)
 end
 
-model = Chain(downsample,u->n_ode(u,25.)[:,:,:,:,1],classify)
+model = Chain(downsample,u->n_ode(u,[25.])[:,:,:,:,end],classify) # Further ugliness getting sol[end]

```

Comment:
on the newest versions of DiffEqFlux it should just give `sol[end]` in this case without the extra dimensions?